util/process_wrapper/utils.h

```diff
@@ -0,0 +1,23 @@
+#ifndef LIB_PROCESS_WRAPPER_UTILS_H_
+#define LIB_PROCESS_WRAPPER_UTILS_H_
+
+#include <string>
+
+#include "util/process_wrapper/system.h"
+
+namespace process_wrapper {
+
+// Converts to and frin the system string format
+System::StrType FromUtf8(const std::string& string);
+std::string ToUtf8(const System::StrType& string);
+
+// Replaces a token in str by replacement
+void ReplaceToken(System::StrType& str, const System::StrType& token,
+                  const System::StrType& replacement);
+
+// Reads a file in text mode and feeds each line to item in the vec output
+bool ReadFileToArray(const System::StrType& file_path, System::StrVecType& vec);
+
```

Comment:
I don't want to add gtest as a dependency, but I can just write a small main for this. Since, the function has coverage through the process_wrapper_tester I though It wouldn't be necessary.

Reply:
That's fair, SGTM.